vpn/vpn.go

```diff
@@ -20,8 +21,8 @@ import (
 
 const (
 	// wireguard should be installed before hand
-	wgManageBin = "wg"
-	wgQuickBin  = "wg-quick"
+	wgManageBin = "sudo wg"
```

Comment:
although it is sudo, user is escalated to use just commands which are whitelisted on serverside.